src/components/home/ventures_section.js

```diff
@@ -58,6 +59,7 @@ const VenturesSection = () => (
           <CallToAction size="large">Let's talk</CallToAction>
         </p>
       </footer>
+      <VenturesPhotos />
```

Comment:
Shouldn't this be above the footer?

Reply:
I'm not sure, I just dropped it here because I thought justo was working on this page.